### PR TITLE
Replace object spread with Object.assign to support older versions of nodejs

### PR DESCRIPTION
**Pull request by GlebkaF**
Hey guys, awesome work! 👍 
Just replaced object spread with Object.assign, because my node does not yet support object spread. 
I guess it might be useful :)

### CODE REVIEW

**Comment by stevemao** [comment]
We support node 8 and 10. So I don't think this is a problem. https://github.com/you-dont-need/You-Dont-Need-Momentjs/blob/master/.travis.yml#L3-L4

**Comment by GlebkaF** [comment]
Earlier versions of node 8, ex. 8.2. doesn’t support Object spread.
It may be problem with something else, but I could not use plugin until I made this change :(

**Comment by stevemao** [comment]
We can probably support the same run times as latest eslint. But since this module is new we think ahead and only support modern node for now. You can always use `babel-node` to run eslint anyways.

My CI runs the following script:

```
babel-node node_modules/eslint/bin/eslint ./
```

**Comment by stevemao** [comment]
I'm open to discussion :) @ddvkid @cht8687 WDYT?

**Comment by ddvkid** [comment]
I'm ok with supporting the earlier version of node with such minor changes.

**Comment by stevemao** [comment]
Ok, can you add the runtime you want to support to travis?

File: lib/index.js [hunk 1]
```diff
@@ -2,9 +2,11 @@ const rules = {
   'no-dynamic-import-moment': require('./rules/no-dynamic-import-moment'),
   'no-import-moment': require('./rules/no-import-moment'),
   'no-moment-constructor': require('./rules/no-moment-constructor'),
-  'no-require-moment': require('./rules/no-require-moment'),
-  ...require('./rules/no-moment-methods'),
+  'no-require-moment': require('./rules/no-require-moment')
 };
+
+Object.assign(rules, require('./rules/no-moment-methods'));
```
Comment (stevemao) [inline lib/index.js]: Don't mutate `rules` here.

**Comment by GlebkaF** [comment]
> Ok, can you add the runtime you want to support to travis?

It seems that I can't specify minor version in travis config

**Comment by stevemao** [comment]
I think you can. just do

```
node_js:
  - '7'
  - '7.2'
  - '6.4.3'
```

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 8
```
Comment (stevemao) [inline .travis.yml]: can you keep this?

**Comment by ddvkid** [comment]
Thanks @GlebkaF !

**Pull request merged**